geonode/layers/templates/layers/layer_leaflet_map.html

```diff
@@ -166,6 +151,7 @@
           onAdd: function(map) {
           var img = L.DomUtil.create('img');
 
+          // I'm not sure what is this logo supposed to be?
           img.src = '../static/img/logo.png';
```

Comment:
Maybe this is for geonode logo?
If that is the case, we can point to geonode logo's static url.